src/main/java/at/hannibal2/skyhanni/config/features/dev/DevConfig.java

```diff
@@ -63,12 +63,20 @@ public class DevConfig {
     @Expose
     @ConfigOption(
         name = "Fancy Contributors",
-        desc = "Marks §cSkyHanni's contributors §7fancy in the tab list. " +
+        desc = "Marks §cSkyHanni's contributors §7fancy in the tab list and nametags. " +
             "§eThose are the folks that coded the mod for you for free :)"
     )
     @ConfigEditorBoolean
     public boolean fancyContributors = true;
 
+    @Expose
+    @ConfigOption(
+        name = "Contributor Nametags",
+        desc = "Makes SkyHanni contributors' nametags fancy too. "
+    )
+    @ConfigEditorBoolean
+    public boolean contributorNametags = false;
```

Comment:
Also @hannibal002 i think this should really be default on, not off

Reply:
I'm not currently fancy, but I feel like having this feature off by default is the same as not having it at all, the only people using it would be friends of contributors and contributors, the feature doesn't really make sense if it's off by default, plus the player already has the option of turning it off, this isn't like sbe where everyone can pay to be obnoxious, this is only for contributors so.
 tldr: if off, no one would turn it on except friends, if on, people who it annoys would turn it off

Reply:
I agree with this argument.
@ItsEmpa feel free to turn it default on.

Reply:
alr, done